sanic/app.py

```diff
@@ -1274,6 +1281,13 @@ async def create_server(
             run_async=return_asyncio_server,
         )
 
+        if self.config.USE_UVLOOP is not _default:
+            error_logger.warning(
+                "You are trying to configure uvloop, but this is only "
+                "supported when using the run(...) method. Sanic will now "
+                "continue to run using the existing event loop."
+            )
```

Comment:
This is a little confusing. It is possible to use `create_server` and `uvloop`. We are inside a coroutine, so it is possible that they already applied the policy. Maybe we should instead mention that the config value is ineffecive.

Reply:
I’ll try to reword this! I did mention “existing event loop” on purpose, but I agree it’s not totally clear.